### PR TITLE
[SQL][Items] Add item mods for items 23087-23091

### PR DESCRIPTION
**Pull request by helixhamin**
Finish some other TODOs and correct some wrong mods.

**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

Adds mods to items 23087-23091.
It also adds some missing mods and corrects one wrong latent effect value.
Note: PFLUG mod ID exists, but the function does not appear to be implemented yet. TRUE_SHOT_EFFECT mod ID exists but the function does not appear to be implemented yet.

## Steps to test these changes

As GM !additem 10730|23087|23088|23089|23090|23091|23129|23330|23464|23665|25962|25963|27473|27474|28207
Equip item and make sure effects take effect (please see note above as not all mods actually work at the moment)

### CODE REVIEW

File: sql/item_latents.sql [hunk 1]
```diff
@@ -35,7 +35,11 @@ INSERT INTO `item_latents` VALUES (10679,11,10,10,0);
 INSERT INTO `item_latents` VALUES (10719,12,10,11,0);
 INSERT INTO `item_latents` VALUES (10719,13,10,11,0);
 INSERT INTO `item_latents` VALUES (10719,14,10,11,0);
-INSERT INTO `item_latents` VALUES (10730,63,10,13,56);  -- WAR AF2 +2 Feet Reduces defense penalty by 10% if Berserk Active
+
+-- -------------------------------------------------------
```
Comment (claywar) [inline sql/item_latents.sql:39]: Just looking at the rest, do we really need these two long lines of comments to create a header here? I'd say let's keep it simple to a single-line comment (and if the rest of the things in latents do that, nuke it from orbit.

**Review by claywar — APPROVED** [review]

File: sql/item_latents.sql (hunk 1 above)
Comment (TeoTwawki) [reply sql/item_latents.sql:39]: They do. let this pass, and later tomorrow I'll mass kill the dashed lines above and below the item name.

This is a case of past conversations everyone forgot about. In the dark days before tooling.

Comment (TeoTwawki) [reply sql/item_latents.sql:39]: we'll start using same format `item_mods` now does after that

Comment (TeoTwawki) [reply sql/item_latents.sql:39]: as promised: #4505

**Pull request merged**